config/main.py

```diff
@@ -5017,6 +5017,7 @@ def bind(ctx, interface_name, vrf_name):
     else:
         config_db.set_entry(table_name, interface_name, {"vrf_name": vrf_name})
 
+    click.echo("Interface {} IPv4 disabled and address(es) removed due to binding VRF {}.".format(interface_name, vrf_name))
```

Comment:
Since both IPv4 and IPv6 addresses are being deleted, both IPv4 and IPv6 routing will be disabled. Can we update this message appropriately?